### PR TITLE
Ida4 master dev ida4 2904

### PR DESCRIPTION
**Pull request by AkshayIntelehealth**

### CODE REVIEW

File: src/app/dashboard/visit-summary/visit-summary.component.scss [hunk 1]
```diff
@@ -754,13 +754,17 @@ th.mat-header-cell:last-child {
...
+  .intel-accordion-title{
+    flex-wrap: nowrap !important;
   }
 
-  .intel-accordion-con .mat-expansion-panel .intel-accordion-title img {
-    width: 30px;
-  }
+  // .intel-accordion-con .mat-expansion-panel .intel-accordion-title h6 {
+  //   font-size: 14px;
+  // }
```
Comment (zeeshan-IH) [inline src/app/dashboard/visit-summary/visit-summary.component.scss:763]: Is this commented code required?

File: src/app/dashboard/visit-summary/visit-summary.component.scss [hunk 2]
```diff
@@ -754,13 +754,17 @@ th.mat-header-cell:last-child {
...
+
+  // .intel-accordion-con .mat-expansion-panel .intel-accordion-title img {
```
Comment (zeeshan-IH) [inline src/app/dashboard/visit-summary/visit-summary.component.scss:765]: This too?

File: src/app/main-container/main-container.component.scss [hunk 3]
```diff
@@ -276,15 +276,15 @@
...
-    }
-
-    img {
-      height: 36px;
-    }
-  }
+  // .header-title {
+  //   h6 {
+  //     font-size: 18px !important;
+  //   }
```
Comment (zeeshan-IH) [inline src/app/main-container/main-container.component.scss:282]: this

**Review by zeeshan-IH — CHANGES_REQUESTED** [review]
@AkshayIntelehealth please check

File: src/app/dashboard/visit-summary/visit-summary.component.scss (hunk 1 above)
Comment (AkshayIntelehealth) [reply src/app/dashboard/visit-summary/visit-summary.component.scss:763]: Previously that was added for PWA app, but as per QA raised issue regarding that so removed 
But in feature if need to revert as that was added previously for specific reason so i just commented the code

Comment (zeeshan-IH) [reply src/app/dashboard/visit-summary/visit-summary.component.scss:763]: If you want certain commented code please add comments above the code, approving it for now; please take care moving forward.

**Review by zeeshan-IH — APPROVED** [review]

**Pull request merged**